Tools/machines/summit-olcf/install_gpu_dependencies.sh

```diff
@@ -94,14 +96,14 @@ python3 -m pip install --upgrade pip
 python3 -m pip install --upgrade virtualenv
 python3 -m pip cache purge
 rm -rf ${SW_DIR}/venvs/warpx-summit
-python3 -m venv ${SW_DIR}/gpu/venvs/warpx-summit
+python3 -m venv ${SW_DIR}/venvs/warpx-summit
 source ${SW_DIR}/venvs/warpx-summit/bin/activate
 python3 -m pip install --upgrade pip
 python3 -m pip install --upgrade wheel
 python3 -m pip install --upgrade cython
 python3 -m pip install --upgrade numpy
 python3 -m pip install --upgrade pandas
-python3 -m pip install --upgrade scipy
+python3 -m pip install --upgrade scipy==1.8.1
```

Comment:
The main reason we use 1.8.1 instead of newer versions is that 1.9+ introduced meson+ninja parallel builds.
We currently found no version to limit the parallelism of these builds, which leads to us getting our build processes killed by the headnode, because it effectively runs with `-j` using all available resources.

Reply:
X-ref: https://github.com/scipy/scipy/issues/18766